rotkehlchen/tests/exchanges/test_kucoin.py

```diff
@@ -763,3 +763,18 @@ def mock_api_query_response(case, options):  # pylint: disable=unused-argument
         )
 
     assert asset_movements == expected_asset_movements
+
+
+@pytest.mark.parametrize('should_mock_current_price_queries', [True])
+def test_query_old_trades_sandbox(sandbox_kuckoin, inquirer):  # pylint: disable=unused-argument
+    """Test that the endpoint for old trades return non invalid results from the api
+    in very old timestamps
+    """
+    trades, _ = sandbox_kuckoin.query_online_trade_history(
+        start_ts=Timestamp(150422400),
```

Comment:
1. Probably you meant `1504224000` as this is the timestamp when kuckoin has launched
2. Maybe use `KUCOIN_LAUNCH_TS` here to be more clear ?